salt/loader_core.py

```diff
@@ -0,0 +1,941 @@
+# -*- coding: utf-8 -*-
```

Comment:
Here's what was left-over from ``salt/loader.py`` after all the ``LazyLoader`` invocations were moved into the sub-system directories to ``.loader.py`` files.